### PR TITLE
Fix Structure Scan Editor Layout

### LINKED ISSUE #10142: UI: Structure Scan module is not structured properly (labels/boxes above one another)
image
Screenshot is pretty self-explanatory.
Labels and modules are getting on top of one another and make thing unusable to setup the Structure Scan.

## Steps to Reproduce:
1. Open latest daily AppImage
2. Start a flight plan and add a structure scan
3. Add a polygon
4. Look at the right colum

## System Information
When posting bug reports, include the following information
- Operating System: Manjaro
- QGC Version: daily
- QGC build: daily Appimage and built from source

## Log Files and Screenshots
```
[D] at ../../qgroundcontrol/qgroundcontrol/src/QGCApplication.cc:281 - "Settings location "/home/rbazile/.config/QGroundControl.org/QGroundControl Daily.ini" Is writable?: true"
[D] at ../../qgroundcontrol/qgroundcontrol/src/QGCLoggingCategory.cc:120 - "Filter rules "*Log.debug=false\nGStreamerAPILog.debug=true\nqt.qml.connections=false""
[D] at ../../qgroundcontrol/qgroundcontrol/src/QGCApplication.cc:384 - "System reported locale: QLocale(English, Latin, United States) ; Name "en_US" ; Preffered (used in maps):  "en-US""
[E] at :0 - "Found no modules in speech-dispatcher. No text to speech possible."
[D] at ../../qgroundcontrol/qgroundcontrol/src/Vehicle/MAVLinkLogManager.cc:360 - "MAVLink logs directory: "/home/rbazile/Documents/QGroundControl Daily/Logs""
[D] at ../../qgroundcontrol/qgroundcontrol/src/QtLocationPlugin/QGCMapEngine.cpp:164 - "Map Cache in: "/home/rbazile/.cache/QGCMapCache300" / "qgcMapCache.db""
[D] at ../../qgroundcontrol/qgroundcontrol/src/MissionManager/MissionController.cc:2370 - "setCurrentPlanViewSeqNum"
[D] at ../../qgroundcontrol/qgroundcontrol/src/MissionManager/MissionController.cc:2370 - "setCurrentPlanViewSeqNum"
[E] at ../../qgroundcontrol/qgroundcontrol/src/VideoReceiver/GstVideoReceiver.cc:284 - "Stop called on empty URI"
[E] at ../../qgroundcontrol/qgroundcontrol/src/VideoReceiver/GstVideoReceiver.cc:284 - "Stop called on empty URI"
[E] at qrc:/qml/PlanView.qml:798 - "qrc:/qml/PlanView.qml:798: TypeError: Cannot read property 'width' of null"
[D] at ../../qgroundcontrol/qgroundcontrol/src/MissionManager/MissionController.cc:2370 - "setCurrentPlanViewSeqNum"
[D] at ../../qgroundcontrol/qgroundcontrol/src/MissionManager/MissionController.cc:2370 - "setCurrentPlanViewSeqNum"
[D] at ../../qgroundcontrol/qgroundcontrol/src/MissionManager/MissionController.cc:2370 - "setCurrentPlanViewSeqNum"
[D] at ../../qgroundcontrol/qgroundcontrol/src/MissionManager/MissionController.cc:2471 - "Found"
[D] at ../../qgroundcontrol/qgroundcontrol/src/MissionManager/MissionController.cc:2479 - "Delayed split"
[D] at ../../qgroundcontrol/qgroundcontrol/src/MissionManager/MissionController.cc:1387 - "Update delayed split segment"
```

### PR DESCRIPTION
**Pull request by holden-zenith**
Fixes #10142

### CODE REVIEW

**Comment by gromain** [comment]
There is an issue with the built AppImage.
Once a polygon is created after adding a structure scan pattern, the module content disappear.
It is correctly formatted while it's still visible though.

**Comment by gromain** [comment]
@holden-zenith is there anything more on my side I could do to help?

**Comment by holden-zenith** [comment]
> @holden-zenith is there anything more on my side I could do to help?

No, I'll take a look at it soon. I've been away

File: src/PlanView/StructureScanEditor.qml [hunk 1]
```diff
@@ -81,50 +79,53 @@ Rectangle {
                     //editorRoot.selectNextNotReadyItem()
                 }
             }
-            */
-        }
+        }*/
```
Comment (patrickelectric) [inline src/PlanView/StructureScanEditor.qml:82]: You can remove the code, it's not necessary to comment if it's not important.

**Review by patrickelectric — CHANGES_REQUESTED** [review]

**Comment by gromain** [comment]
Works for me

**Review by patrickelectric — APPROVED** [review]

**Comment by mrpollo** [comment]
thanks @holden-zenith

**Pull request merged**